scripts/exglobal_prep_ocean_obs.py

```diff
@@ -4,15 +4,19 @@
 from datetime import datetime, timedelta
 from multiprocessing import Process
 import os
+from pathlib import Path
 import subprocess
 from soca import prep_marine_obs
 from wxflow import YAMLFile, save_as_yaml, FileHandler, Logger
+from wxflow import add_to_datetime, to_timedelta, datetime_to_YMDH
```

Comment:
this should be combined with the above line - I think it can be broken out into multiple lines if you put the imported packages into parens like so (courtesy of ChatGPT):
from X import (
    Y1,
    Y2,
    Y3
)